testng-curl/README.md

```diff
@@ -0,0 +1,100 @@
+# testng-browserstack
+
+[TestNG](http://testng.org) Integration with BrowserStack.
+
+![BrowserStack Logo](https://d98b8t1nnulk5.cloudfront.net/production/images/layout/logo-header.png?1469004780)
+
+## Using Maven
+
+### Run sample build
+
+- Clone the repository
+- Replace YOUR_USERNAME and YOUR_ACCESS_KEY with your BrowserStack access credentials in browserstack.yml.
+- Install dependencies `mvn compile`
+- To run the test suite having cross-platform with parallelization, run `mvn test -P sample-test`
+- To run local tests, run `mvn test -P sample-local-test`
+
+Understand how many parallel sessions you need by using our [Parallel Test Calculator](https://www.browserstack.com/automate/parallel-calculator?ref=github)
+
+### Integrate your test suite
+
+This repository uses the BrowserStack SDK to run tests on BrowserStack. Follow the steps below to install the SDK in your test suite and run tests on BrowserStack:
```

Comment:
Reason for using BrowserStack SDK ?
It adds a dependency on SDK side and gives me an overall impression that code cannot be triggered without SDK.
Running on Automate product is a good choice by asking for username and access key, but having a dependency on SDK inside cURL folding is something which worries me

If any genuine use, please comment

Reply:
later